src/snowflake/snowpark/_internal/analyzer/table_function.py

```diff
@@ -71,6 +71,13 @@ def __init__(
         self.args = args
 
 
+class GeneratorTableFunction(TableFunctionExpression):
+    def __init__(self, args: Dict[str, Expression], operators: List) -> None:
+        super().__init__("generator")
```

Comment:
Are there ways to add `partition_spec` for generator functions?

Reply:
do we need it? generator function doesn't have partition_spec?